PPI/manual_work/Jul_2023/manual_tables.sql

```diff
@@ -0,0 +1,383 @@
+       cr.valid_start_date,
+       current_date as valid_end_date,
+       'D' as invalid_reason
+FROM concept c
+LEFT JOIN concept_relationship cr on c.concept_id = cr.concept_id_1
+LEFT JOIN concept c1 on cr.concept_id_2 = c1.concept_id
+WHERE c.concept_code in (SELECT concept_code FROM concept_manual)
+AND cr.relationship_id not in ('Question source of', 'Mapped from', 'Answer of (PPI)', 'PPI parent code of'));
+
+--insert new concepts into concept_manual
+--insert module bhp
+INSERT INTO concept_manual
+SELECT DISTINCT
+'Behavioral Health and Personality' AS concept_name,
+'Observation' AS domain_id,
+'PPI' AS vocabulary_id,
+'Module' AS concept_class_id,
+'S' as source_standard_concept,
+'bhp' as concept_code,
+CURRENT_DATE AS valid_start_date,
+TO_DATE('20991231','yyyymmdd') AS valid_end_date,
+NULL AS invalid_reason
+;
+
+-- insert questions bhp
+INSERT INTO concept_manual
+SELECT DISTINCT
+question_name AS concept_name,
+'Observation' AS domain_id,
+'PPI' AS vocabulary_id,
+'Question' AS concept_class_id,
+'S' as source_standard_concept,
+question_code as concept_code,
+CURRENT_DATE AS valid_start_date,
+TO_DATE('20991231','yyyymmdd') AS valid_end_date,
+NULL AS invalid_reason
+from bhp_pr;
+--where lower(question_code) not in (select lower(concept_code) from concept_manual) ;
+
+--insert answers bhp
+INSERT INTO concept_manual
+SELECT DISTINCT
+answer_name AS concept_name,
+'Observation' AS domain_id,
+'PPI' AS vocabulary_id,
+'Answer' AS concept_class_id,
+null AS source_standard_concept,
+answer_code AS concept_code,
+CURRENT_DATE AS valid_start_date,
+TO_DATE('20991231','yyyymmdd') AS valid_end_date,
+NULL AS invalid_reason
+from bhp_pr;
+
+--insert module ehh
+INSERT INTO concept_manual
+SELECT DISTINCT
+'Emotional Health History and Well-Being' AS concept_name,
+'Observation' AS domain_id,
+'PPI' AS vocabulary_id,
+'Module' AS concept_class_id,
+'S' as source_standard_concept,
+'ehhwb' as concept_code,
+CURRENT_DATE AS valid_start_date,
+TO_DATE('20991231','yyyymmdd') AS valid_end_date,
+NULL AS invalid_reason
+;
+
+-- insert questions ehh
+INSERT INTO concept_manual
+SELECT DISTINCT
+question_name AS concept_name,
+'Observation' AS domain_id,
+'PPI' AS vocabulary_id,
+'Question' AS concept_class_id,
+'S' as source_standard_concept,
+question_code as concept_code,
+CURRENT_DATE AS valid_start_date,
+TO_DATE('20991231','yyyymmdd') AS valid_end_date,
+NULL AS invalid_reason
+from ehh_pr ;
+--where lower(question_code) not in (select lower(concept_code) from concept_manual) ;
+
+--insert answers ehh
+INSERT INTO concept_manual
+SELECT DISTINCT
+answer_name AS concept_name,
+'Observation' AS domain_id,
+'PPI' AS vocabulary_id,
+'Answer' AS concept_class_id,
+null AS source_standard_concept,
+answer_code AS concept_code,
+CURRENT_DATE AS valid_start_date,
+TO_DATE('20991231','yyyymmdd') AS valid_end_date,
+NULL AS invalid_reason
+from ehh_pr;
+
+--insert questions-answers relationships into concept_relationship_manual
```

Comment:
Why is this part commented?

Reply:
it was done for reverse relationships.
now resolved.